test/test_cache.py

```diff
@@ -242,38 +249,43 @@ def test_load_by_guid(self, mock_load_quick, mock_load):
 
         with self.subTest("PM-only"):
             cache = Cache(self.gc, "GC6MKEF", guid="53d34c4d-12b5-4771-86d3-89318f71efb1")
-            with self.assertRaises(PMOnlyException):
-                cache.load_by_guid()
+            with recorder.use_cassette('cache_guidload_PMO'):
+                with self.assertRaises(PMOnlyException):
+                    cache.load_by_guid()
 
         with self.subTest("calls load_quick if no guid"):
             cache = Cache(self.gc, "GC2WXPN")
-            with self.assertRaises(Exception):
-                cache.load_by_guid()  # Raises error since we mocked load_quick()
+            with recorder.use_cassette('cache_guidload_fallback'):
+                with self.assertRaises(Exception):
+                    cache.load_by_guid()  # Raises error since we mocked load_quick()
             self.assertTrue(mock_load_quick.called)
 
     def test_load_trackables(self):
         cache = Cache(self.gc, "GC26737")  # TB graveyard - will surelly have some trackables
-        trackable_list = list(cache.load_trackables(limit=10))
+        with recorder.use_cassette('cache_trackables'):
+            trackable_list = list(cache.load_trackables(limit=10))
         self.assertTrue(isinstance(trackable_list, list))
 
     def test_load_logbook(self):
-        log_authors = list(map(lambda log: log.author, self.c.load_logbook(limit=200)))  # limit over 100 tests pagging
+        with recorder.use_cassette('cache_logbook'):
+            # limit over 100 tests pagging
```

Comment:
pagination pls, sorry

Reply:
Fixed in c1882da86de3efb31ff395d60c4b08a512c4eed7.